src/Microsoft.Azure.SignalR.Management/RestApiProvider.cs

```diff
@@ -16,27 +16,26 @@ internal class RestApiProvider
 
         private readonly RestApiAccessTokenGenerator _restApiAccessTokenGenerator;
 
-        private readonly string _audienceBaseUrl;
-        private readonly string _serverEndpoint;
+        private readonly ServiceEndpoint _serviceEndpoint;
+
+        private string AudienceBaseUrl => _serviceEndpoint.AudienceBaseUrl;
+
+        private AuthType AuthType => _serviceEndpoint.AccessKey.AuthType;
+
+        private string ServerEndpoint => _serviceEndpoint.ServerEndpoint.AbsoluteUri;
 
         public RestApiProvider(ServiceEndpoint endpoint)
         {
-            _audienceBaseUrl = endpoint.AudienceBaseUrl;
-            _serverEndpoint = endpoint.ServerEndpoint.AbsoluteUri;
+            _serviceEndpoint = endpoint;
             _restApiAccessTokenGenerator = new RestApiAccessTokenGenerator(endpoint.AccessKey);
         }
 
-        private string GetPrefixedHubName(string applicationName, string hubName)
```

Comment:
Resorting the methods make it difficult to find out what is the actual functional change.